curvlinops/kfac_utils.py

```diff
@@ -0,0 +1,92 @@
+"""Utility functions related to KFAC."""
+
+from math import sqrt
+from typing import Union
+
+from torch import Tensor, diag, einsum, eye
+from torch.nn import CrossEntropyLoss, MSELoss
+
+
+def hessian_matrix_sqrt(
```

Comment:
Could be a bit more explicitly named as `loss_hessian_matrix_sqrt`.

Reply:
Agree. Done.